app/src/components/common/Header.tsx

```diff
@@ -18,7 +18,14 @@ import {useAuth} from '../../app/hooks/useAuth';
 import {useSignOutMutation} from '../../services/auth';
 import {useAppDispatch} from '../../app/hooks/store';
 import {setCredentials} from '../../app/authSlice';
-import {useNavigate} from 'react-router-dom';
+import {Link, useNavigate} from 'react-router-dom';
+
+const userName = 'Hank Hill';
+
+function getInitials(name: string) {
+  const splitName = name.split(' ');
+  return splitName[0][0] + splitName[1][0];
```

Comment:
This should have more checks. For example, check if `splitName` has more than 1 element.

Reply:
Thanks! Added a check for length and also to use the last element in case it's not just first and last name.